Alternatives section in README (#185)
rs-ipfs/rust-ipfs: README.md

Rust IPFS is actively maintained by @koivunej, and @aphelionz. Special thanks is given to [Protocol Labs](https://github.com/protocol), [Equilibrium Labs](https://github.com/eqlabs), and [MRH.io](https://mrh.io).

## Alternatives and other cool, related projects

> It’s been noted that the Rust-IPFS name and popularity may serve us from a "first-mover" perspective.  However, alternatives do exist with different philosophies, and we believe that supporting this community and diversity is both important and will ultimately help produce the best solution possible.

Reviewer: ```suggestion
It’s been noted that the Rust-IPFS name and popularity may serve its organization from a "first-mover" perspective. However, alternatives with different philosophies do exist, and we believe that supporting a diverse IPFS community is important and will ultimately help produce the best solution possible.
```
